cpp/cmake/thirdparty/get_cucollections.cmake

```diff
@@ -22,11 +22,13 @@ function(find_and_configure_cucollections)
 
     # Find or install cuCollections
     CPMFindPackage(NAME   cuco
-        GITHUB_REPOSITORY PointKernel/cuCollections
-        GIT_TAG           static-multi-map
-        OPTIONS           "BUILD_TESTS OFF"
-                          "BUILD_BENCHMARKS OFF"
-                          "BUILD_EXAMPLES OFF"
+        GLOBAL_TARGETS cuco::cuco
```

Comment:
GLOBAL_TARGETS and CPM_ARGS are features of `rapids-cmake` and the `rapids_cpm_find` function. 

With https://github.com/rapidsai/cudf/pull/9030 being merged you should change `CPMFindPackage` to `rapids_cpm_find`.

Reply:
Done.